### DISCUSSION TITLE
Help ! Unknown Type

### DISCUSSION THREAD

**Discussion by OscarDelta59** [MahApps/MahApps.Metro#4453] [Q&A]
Hi there,

I'm a Newbee using VB, WPF apps and Mahapps. I need to create some powershell scripts (no problem with powershell), and want to offer a Modern GUI ...

I've followed the How to guide.

Everything is ok in VB :

![VB wpf](https://github.com/MahApps/MahApps.Metro/assets/154590070/02781da6-cee6-4b97-bb0d-9194d463536a)

However, using Powershell, I get this error :

Exception lors de l'appel de « Load » avec « 1 » argument(s) : « Impossible de créer le type inconnu '{[http://metro.mahapps.com/winfx/xaml/controls}MetroWindow'. »](http://metro.mahapps.com/winfx/xaml/controls%7DMetroWindow'.%C2%A0%C2%BB)

I've googled it for hours, without success.

The PS1 file :

`clear-host

[System.Reflection.Assembly]::LoadFrom('C:_TestAppWpf\Metro\MahApps.Metro.dll') | out-null
[System.Reflection.Assembly]::LoadFrom('C:_TestAppWpf\Metro\System.Windows.Interactivity.dll') | out-null
[System.Reflection.Assembly]::LoadFrom('C:_TestAppWpf\Metro\ControlzEx.dll') | out-null

function LoadXml ($FileName) {

$XamlLoader = (new-object system.Xml.XmlDocument)
$XamlLoader.Load($FileName)
return $XamlLoader
}

$XamlWindow = LoadXml("C:_TestAppWpf\test.xaml")
$Reader = (New-Object System.Xml.XmlNodeReader $XamlWindow)
$Form = [Windows.Markup.XamlReader]::Load($Reader)

$Form.ShowDialog()`

The XAML file :

`<Controls:MetroWindow x:Class="MainWindow"
xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
xmlns:Controls="http://metro.mahapps.com/winfx/xaml/controls"
xmlns:local="clr-namespace:Test"
mc:Ignorable="d"
Title="Test App Wpf" Height="450" Width="800">

<Window.Resources>
    <ResourceDictionary>
        <ResourceDictionary.MergedDictionaries>
            
            <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Controls.xaml" />
            <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Fonts.xaml" />
            <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Colors.xaml" />
            
            
            <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Accents/Yellow.xaml" />
           
           <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Accents/BaseDark.xaml" />1
        </ResourceDictionary.MergedDictionaries>
    </ResourceDictionary>
</Window.Resources>

<Grid>
    <Label Name="label" Content="Client" HorizontalAlignment="Left" Margin="10,10,0,0" VerticalAlignment="Top" Width="95"/>
    <TextBox Name="textBox" HorizontalAlignment="Left" Margin="10,36,0,0" TextWrapping="Wrap" Text="TextBox" VerticalAlignment="Top" Width="222"/>
    <Button Name="button" Content="Start" HorizontalAlignment="Left" Margin="10,76,0,0" VerticalAlignment="Top" RenderTransformOrigin="-1.667,0.055" Width="222"/>

</Grid>
</Controls:MetroWindow>
`

 Versions : Windows 11 22H2, Visual Studio  17.5.1, Mahapps Metro 2.4.10

Thank you for your help.

Kind regards,
Oliver.

**Comment by jeffshumphreys** [comment]
What's your error? I'm getting: | Exception calling "ShowDialog" with "0" argument(s): "Cannot create **unknown type** '{http://metro.mahapps.com/winfx/xaml/iconpacks}PackIconMaterial'."

**Comment by timunie** [comment]
IconPacks is installed I guess?
